### PR TITLE
Transcripts add temporary player item to see transcripts

### PR DESCRIPTION
**Pull request by SergioEstevao**
Refs #1848 

This PR adds a temporary tab view on the player to show a transcripts if available.

## To test

- Start the app
- Ensure that you have the `transcripts` and `newShowNotesEndpoint` FF enabled
- Open an episode with Transcripts ( Cautionary Tales podcast episode have translations)
- Play the episode
- Open the Full Screen Player
- Open the tab for Transcripts
- See if you can see the transcript and scroll around.
- Check that when the episode finishes and a new episode is loaded the transcripts updates correctly.

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: podcasts/PlayerTabsView.swift [hunk 1]
```diff
@@ -21,6 +22,8 @@ enum PlayerTabs: Int {
             return L10n.chapters
         case .bookmarks:
             return L10n.bookmarks
+        case .transcripts:
+            return "Transcripts"
```
Comment (SergioEstevao) [inline podcasts/PlayerTabsView.swift]: I'm not localising the strings on this PR because these are temporary until we have final designs.

Comment (leandroalonso) [reply podcasts/PlayerTabsView.swift]: I believe we can start doing so, unless is a very specific copy? "Transcripts" I'm sure we will have in the final version somehow.

Anyway, not a strong feeling.

**Review by leandroalonso — APPROVED** [review]
Tested with the flag enabled and disabled and it works as expected.

@SergioEstevao one logic we might want to add is to display the tab only if a transcript is ready. This is a logic we will need anyway, independently of design: if there are transcripts we will show a tab/card/button/something, if not, we won't show.

However, I don't feel this needs to be tackled now, just something for us to keep in mind (and this is kinda done in the transcripts VC).

**Comment by SergioEstevao** [comment]
> However, I don't feel this needs to be tackled now, just something for us to keep in mind (and this is kinda done in the transcripts VC).

I will add this on the follow up PR

**Pull request merged**